components/bl-credit-card-component/src/helpers.js

```diff
@@ -0,0 +1,312 @@
+const eloStringPattern = '^401178|^401179|^431274|^438935|^451416|^457393|^457631|^457632|^504175|^627780|^636297' +
```

Comment:
if it comes from a library please add a link to the lib for reference

Reply:
It comes from the [Payment ](https://github.com/jessepollak/payment/blob/master/src/index.coffee) library.